app/views/admin/reports/_actions.html.haml

```diff
@@ -17,21 +17,27 @@
       .report-actions__item__button
         = form.button t('admin.reports.delete_and_resolve'),
                       name: :delete,
-                      class: 'button button--destructive'
+                      class: 'button button--destructive',
+                      disabled: statuses.empty?,
+                      title: statuses.empty? ? t('admin.reports.actions_no_posts') : ''
```

Comment:
I think this could entirely be skipped when there are no posts selected.

Reply:
I asked a few mods and they said they'd rather the buttons be there consistently but disabled, and I think I understand that, since it makes things predictable